new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiSample.java

```diff
@@ -0,0 +1,244 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyOperator;
+import static oracle.weblogic.kubernetes.utils.TestUtils.getDateAndTimeStamp;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Tests to verify MII sample.
+ */
+@DisplayName("Test model in image sample")
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@IntegrationTest
+public class ItMiiSample implements LoggedTest {
+
+  private static final String MII_SAMPLES_WORK_DIR = WORK_DIR
+      + "/model-in-image-sample-work-dir";
+  private static final String MII_SAMPLES_SCRIPT =
+      "../src/integration-tests/model-in-image/run-test.sh";
+
+  private static String DOMAIN_TYPE = "WLS";
+  private static String MII_SAMPLE_IMAGE_NAME1 = REPO_NAME + "mii-" + getDateAndTimeStamp();
+  private static String MII_SAMPLE_IMAGE_NAME2 = REPO_NAME + "mii-" + getDateAndTimeStamp();
+  private static String MII_SAMPLE_IMAGE_TAG_V1 = DOMAIN_TYPE + "-v1";
+  private static String MII_SAMPLE_IMAGE_TAG_V2 = DOMAIN_TYPE + "-v2";
+  private static String imageNameV1 = MII_SAMPLE_IMAGE_NAME1 + ":" + MII_SAMPLE_IMAGE_TAG_V1;
+  private static String imageNameV2 = MII_SAMPLE_IMAGE_NAME2 + ":" + MII_SAMPLE_IMAGE_TAG_V2;
+  private static String SUCCESS_SEARCH_STRING = "Finished without errors";
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static String traefikNamespace = null;
+  private static Map<String, String> envMap = null;
+
+  /**
+   * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *        JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
+
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Creating unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    logger.info("Creating unique namespace for Treafik");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    traefikNamespace = namespaces.get(2);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // env variables to override default values in sample scripts
+    envMap = new HashMap<String, String>();
+    envMap.put("DOMAIN_NAMESPACE", domainNamespace);
+    envMap.put("TRAEFIK_NAMESPACE", traefikNamespace);
+    envMap.put("WORKDIR", MII_SAMPLES_WORK_DIR);
+    envMap.put("MODEL_IMAGE_NAME", MII_SAMPLE_IMAGE_NAME1);
+    envMap.put("MODEL_DIR", "model-images/model-in-image__" + MII_SAMPLE_IMAGE_TAG_V1);
```

Comment:
It should not be necessary to set MODEL_DIR anymore.

Reply:
removed